gnss_lib_py/algorithms/multi_gnss.py

```diff
@@ -0,0 +1,493 @@
+"""Functions to process multi-GNSS related files.
+
+"""
+
+__authors__ = "Sriramya Bhamidipati"
+__date__ = "05 August 2022"
+
+from datetime import datetime, timedelta, timezone
+
+import numpy as np
+from scipy import interpolate
+
+from gnss_lib_py.utils.sim_gnss import find_sat
+from gnss_lib_py.parsers.ephemeris import EphemerisManager
+import gnss_lib_py.utils.constants as consts
+from gnss_lib_py.utils.time_conversions import gps_millis_to_tow
+
+def extract_sp3_func(sp3data, sidx, ipos = 10, \
```

Comment:
Are these top two functions expected to be called outside of this function (e.g. by users?) if not, they should be private functions noted with an underscore in the front of their name and be at the bottom of the file.

Reply:
except compute_multi_gnss_from_precise_eph none of the functions in precise_ephemerides.py are public.